news/2023/2023-11-03-interview-with-ephemeral.md

```diff
@@ -0,0 +1,104 @@
+
+![](meow)
+
+On October 28th, the [osu! Medal Hunters](https://discord.gg/8qpNTs6) Discord server staff hosted an interview with [Ephemeral](https://osu.ppy.sh/users/102335), and this newspost will highlight some of the best questions that were asked here! If you wish to view the recording of the full event, you may do so [here](https://youtu.be/OHWbFpPoTJI).
+
+And so, without further hesitations, let's dive right into it, starting up with a small introduction:
+
+**Who are you?**
+
+I am some guy named Ephemeral, who has been doing osu! things for like, 14 years(?)
+
+**How and why did you join the osu!team?**
+
+Once upon a time, there was some guy and he made a very bad map, insisting that his map was good. I saw this map and for some reason their attitude really irked me, so I posted what I guess would be the equivalent of a "mania mod post" in 2009. That kind of got me infamous in the community, because I tore his map to bits, and then I think six months later I was nominated to the [BAT](/wiki/People/Beatmap_Appreciation_Team) at the time, so that was where it all kind of began. 
+I did BAT work for about six months and then decided I hated it, so I just started doing moderation stuff instead, and then I kind of just stuck around and helped out with a whole bunch of early teams, like the [MAT](/wiki/People/Mapping_Assistance_Team).
+
+**So how did you start doing medal stuff?**
+
+I recall vaguely that peppy mentioned that he'd like some more achievements, but doesn't know what to do, so I was like: "you know, I got some ideas, I'll send them to you in PM." I sent them to him in PM and said like: "hey, these are pretty good, do you want to do some more?", and I said ok, I'll do some more. 
+Then he sent me the spreadsheet they were using for them, and I think I started to kind of tidy it up, because it was all a bit scuffed, as it was just a really early prototype of medals. 
+So I think at that point I started getting them in order, and once we had them all in order we ported first generation of medals to the new system, which was called osu!next at the time. The next website started showing the medals properly, and so we started with the Mod Introduction medals and stuff like that. So I've been around involved in medals since then basically.
+
+**What are your thoughts on the current state of medals?**
+
+Overall I think the past couple of releases, the last release in particular came out fantastic, I'm very happy with it. 
+I think they are getting better, but I think with operating at like maybe 20% of what medals will ultimately end up being when they're fully finished being implemented in lazer, because it is very limited what we can do with them at the moment. And lazer will pull back a lot of those limitations and criteria that realistically would not be possible in stable.
```

Comment:
he's talking about limitations on what things can be checked for medal requirements, but it's assumed that the reader already knows of such things at all

Reply:
Honestly not sure what to do about this, maybe something like "Writer's note" would work? Feel free to suggest other things you have in mind

Reply:
Still need to think of a good way to fix this one